src/js/tools/pen.js

```diff
@@ -66,25 +66,31 @@ define(function (require, exports, module) {
                     return descriptor.playObject(optionsObj);
                 });
 
-        if (this.vectorMode) {
-            var selectVectorMask = descriptor.playObject(vectorMaskLib.activateVectorMaskEditing());
-            return Promise.join(setPromise, selectVectorMask);
+        var deleteFn = function (event) {
+            event.stopPropagation();
+
+            return PS.performMenuCommand(_CLEAR_PATH)
+                .catch(function () {
+                    // Silence the errors here
+                });
+        };
+
+        // Disable target path suppression
```

Comment:
Can you move this comment to the right place, or alternatively leave it here but add a note about what the first two shortcuts are for? Also, would it be better to call `shortcuts.addShortcuts` once instead? Finally, don't these delete/backspace shortcuts need to be removed when the pen tool is deselected?